archive/feature-specifications/README.md

```diff
@@ -0,0 +1,33 @@
+## Background material about features
+
+This directory contains background material only. Consult the 
+[language specification](https://dart.dev/guides/language/spec)
+and the 
+[language repository](https://github.com/dart-lang/language)
+for current information.
+
+### What is a feature specification?
+
+In order to move faster and get better feedback, we implement and iterate on
+language changes before the full official specification has been written.
+Still, the implementers need *something* to go on.
+
+For that, the language team writes _feature specifications_. These are
+intended to be precise enough for a good faith implementer to correctly
+understand the syntax and semantics of the language, but when the contents
+of a feature specification is integrated into the language specification we
+expect the extra processing to give rise to additional clarifications and
+corrections, which means that a feature specification is expected to be 
+_nearly_ as complete and correct as the language specification.
+
+### Feature specifications in this directory
+
+This directory contains older feature specifications, newer ones can be
+found in, respectively should be submitted to, the language repository
+[here](https://github.com/dart-lang/language).
```

Comment:
May want to adjust or remove this paragraph now that it lives in the language repo.

Reply:
Done.